helix-term/src/health.rs

```diff
@@ -300,6 +300,8 @@ pub fn language(lang_str: String) -> std::io::Result<()> {
         lang.debugger.as_ref().map(|dap| dap.command.to_string()),
     )?;
 
+    probe_parser(lang.grammar.as_ref().unwrap_or(&lang.language_id))?;
```

Comment:
I think this would make sense below the formatter instead of above so it's right next to the checks/x's for the queries